examples/profile/views/pages/index.ejs

```diff
@@ -64,16 +67,31 @@
   </main>
   <script src="https://www.yoti.com/share/client/"></script>
   <script>
-  window.Yoti.Share.init({
-    "elements": [{
-      "domId": "yoti-share-button",
-      "clientSdkId": "<%= yotiClientSdkId %>",
-      "scenarioId": "<%= yotiScenarioId %>",
-      "button": {
-        "label": "Use Yoti"
-      }
-    }]
-  });
+    window.Yoti.Share.init({
+
+
+      "elements":
+        [
+          {
+            "domId": "yoti-share-button",
+            "scenarioId": "<%= yotiScenarioId %>",
+            "clientSdkId": "<%= yotiClientSdkId %>",
+            "button":
+            {
+              "label": "Use Yoti",
+              "align": "center",
+              "width": "auto",
+              "verticalAlign": "top"
+            },
+            "type": "modal",
+            "qr":
+            {
+              "title": "Scan with the Yoti app:"
+            }
+          }
+        ]
+
+    });
```

Comment:
`qr.title` doesn't affect the modal QR title, perhaps we should remove it?

We could also remove the whitespace here

```suggestion
    window.Yoti.Share.init({
      "elements": [
        {
          "domId": "yoti-share-button",
          "scenarioId": "<%= yotiScenarioId %>",
          "clientSdkId": "<%= yotiClientSdkId %>",
          "button": {
            "label": "Use Yoti",
            "align": "center",
            "width": "auto",
            "verticalAlign": "top"
          },
          "type": "modal"
        }
      ]
    });
```